modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/DdlWithMockedManagersTest.java

```diff
@@ -156,11 +163,56 @@ void before() throws NodeStoppingException {
         queryProc.start();
     }
 
+    /**
+     * Checks that appropriate methods called form table manager on rel node construction.
+     */
+    @Test
+    void checkAppropriateTableFound() throws Exception {
```

Comment:
how this test relates to DDL operations?

Reply:
rename whole test